lib/spack/docs/environments.rst

```diff
@@ -1000,6 +1000,28 @@ For example, the following environment has three root packages:
 This allows for a much-needed reduction in redundancy between packages
 and constraints.
 
+-------------------------------
+Modifying Environment Variables
+-------------------------------
+
+Spack Environments can modify environment variables when activated.  The environment can be
+configured to set, unset, prepend, or append using ``env_vars`` option in the ``spack.yaml``
```

Comment:
```suggestion
configured to set, unset, prepend, or append using ``env_vars`` configuration in the ``spack.yaml``  or through config scopes
```
This is a configuration scope, and users can set the values in `env_vars.yaml` as well.